Automattic/sensei
# Fix undo in module blocks

### Changes proposed in this Pull Request

* This approach came up from [this PR](https://github.com/Automattic/sensei/pull/3801). You can have a look there for some context.
* The previous approach for sharing the border setting between the module and the outline was breaking undo because `setAttributes` was used.
* In the new approach, `setAttributes` is only called as part of user interactions and undo functionality is not broken.
* When updating the border setting in the outline level, the setting is applied in all modules.

### Testing instructions

* Insert an outline module and add some modules.
* Play around with the module border settings.
* Observe that the border is displayed correctly in both frontend and editor.
